setup.py

```diff
@@ -45,10 +47,10 @@ def get_requirements():
     url="https://github.com/jalam453/pubtools-content-gateway",
     classifiers=classifiers,
     packages=find_packages(exclude=["tests"]),
-    install_requires=get_requirements(),
+    install_requires=INSTALL_REQUIRES,
```

Comment:
Pls utilize the requirements file instead of specifying the dependencies in setup.py

Reply:
I am getting some issues while using requirements from files so I have mentioned the dependencies here in the setup file. Will be using the dependencies from the requirements file only but later in the development phase.